### PR TITLE
doc: improve prometheus docs

### PR DESCRIPTION
**Pull request by Janik-Haag**
Adding docs about how to read the bearer token from a file

### CODE REVIEW

File: docs/docs/30-administration/90-prometheus.md [hunk 1]
```diff
@@ -30,6 +30,20 @@ scrape_configs:
        - targets: ['woodpecker.domain.com']
 ```
 
+You can also read the token from a file like so:
```
Comment (xoxys) [inline docs/docs/30-administration/90-prometheus.md:33]:
```suggestion
As an alternative, the token can also be read from a file:
```

File: docs/docs/30-administration/90-prometheus.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Prometheus
 
-Woodpecker is compatible with Prometheus and exposes a `/metrics` endpoint. Please note that access to the metrics endpoint is restricted and requires an authorization token with administrative privileges.
+Woodpecker is compatible with Prometheus and exposes a `/metrics` endpoint if you set the `WOODPECKER_PROMETHEUS_AUTH_TOKEN` environment variable. Please note that access to the metrics endpoint is restricted and requires an authorization token with administrative privileges.
```
Comment (xoxys) [inline docs/docs/30-administration/90-prometheus.md:3]:
```suggestion
Woodpecker is compatible with Prometheus and exposes a `/metrics` endpoint if the environment variable `WOODPECKER_PROMETHEUS_AUTH_TOKEN` is set. Please note that access to the metrics endpoint is restricted and requires an authorization token with administrative privileges.
```

**Review by xoxys — CHANGES_REQUESTED** [review]
Thanks! Only small adjustments in the wording, the rest looks good.

**Comment by Janik-Haag** [comment]
Should be good now?

**Review by xoxys — APPROVED** [review]

**Pull request merged**